docs/src/filters/writing_custom_filters.md

```diff
@@ -392,14 +387,13 @@ However, it usually contains a Protobuf equivalent of the filter's static config
        }
        ```
 
-[Filter]: #
-[FilterFactory]: #
-[filter-factory-name]: #FilterFactory::name
-[FilterRegistry]: #
-[FilterChain]: #
-[runner]: #
-[create-filter-args-config]: #CreateFilter::config
-[config-type-dynamic]: #ConfigType::Dynamic
+[Filter]: https://docs.rs/quilkin/{{version}}/quilkin/filters/trait.Filter.html
+[FilterFactory]: https://docs.rs/quilkin/0.1.0/quilkin/filters/trait.FilterFactory.html
+[filter-factory-name]: https://docs.rs/quilkin/0.1.0/quilkin/filters/trait.FilterFactory.html#tymethod.name
+[FilterRegistry]: https://docs.rs/quilkin/0.1.0/quilkin/filters/struct.FilterRegistry.html
+[runner::run]: https://docs.rs/quilkin/0.1.0/quilkin/runner/fn.run.html
+[CreateFilterArgs::config]: https://docs.rs/quilkin/0.1.0/quilkin/filters/prelude/struct.CreateFilterArgs.html#structfield.config
+[ConfigType::dynamic]: https://docs.rs/quilkin/0.1.0/quilkin/filters/enum.ConfigType.html#variant.Dynamic
```

Comment:
This is why I think we should move this stuff to rustdoc, it would always go to the correct version.

Reply:
We can also just use relative links to our local versions of the API docs - since they are published together.

Reply:
They aren't published together at mdbook build/test time though, so that would fail any linkchecking.

Reply:
We could also use a seperate link checker - we don't have to be tied just to mdbook's link checker - then we can check links across the entire documentation set. This will be particularly useful as we will probably end up with links going back and forth between rustdoc and mdbook.

I'm particularly partial to https://github.com/wjdp/htmltest for link checking across an entire website.

Reply:
Created https://github.com/googleforgames/quilkin/issues/367 to track the discussion!